src/pages/stacks-blockchain/overview.md

```diff
@@ -89,7 +51,24 @@ An example of this is Stacking.
 
 ### Stacking
 
-Stackers are Stacks (STX) Token holders who provide a valuable service to the network by locking up their STX for a certain
+Stackers are Stacks (Stacks) Token holders who provide a valuable service to the network by locking up their Stacks for a certain
 period of time. As a reward, Stackers receive the Bitcoin that miners transfer as part of Proof of Transfer (PoX).
 
 -> Proof of Transfer and Stacking are in active development and are coming soon
+
+## Design specification
```

Comment:
[nit] "Design specification" as a title sounds odd, esp. as the first line says "describing the implementation". I would just say "Protocol Specification" instead.